### PR TITLE
:bug:: remove managedclustersetbinding v1beta1 in webhook.

### PR DESCRIPTION
**Pull request by xuezhaojun**
## Summary

We have split the webhook configuration of managedclustersetbinding to 2 configs in this PR: https://github.com/open-cluster-management-io/registration-operator/pull/293

The previous release are not affected and all functionalities are fine, just remove v1beta1 content to make versions more clear.

## Related issue(s)

Fixes #

### CODE REVIEW

File: manifests/cluster-manager/hub/cluster-manager-registration-webhook-clustersetbinding-validatingconfiguration.yaml [hunk 1]
```diff
@@ -22,6 +22,6 @@ webhooks:
     - v1beta2
     resources:
     - managedclustersetbindings
-  admissionReviewVersions: ["v1beta1","v1"]
+  admissionReviewVersions: ["v1beta2","v1"]
```
Comment (zhujian7) [inline manifests/cluster-manager/hub/cluster-manager-registration-webhook-clustersetbinding-validatingconfiguration.yaml:25]: seems the [admissionReviewVersions](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#webhook-request-and-response) is not related to the to-be-validated resources version? Not sure if we should change this.

Comment (xuezhaojun) [reply manifests/cluster-manager/hub/cluster-manager-registration-webhook-clustersetbinding-validatingconfiguration.yaml:25]: In fact, this list is not used in our customized webhook logic,  the only field we leverage is UserInfo: https://github.com/open-cluster-management-io/ocm/blob/6571a8a4d9aadae5014c9243a5acdb3667e4758f/pkg/registration/webhook/v1beta2/managedclustersetbinding_validating.go#L39

So in this configuration, we can put any version for example "v2".  We change it to v1beta2 just to aglin with our version migration process.

Comment (xuezhaojun) [reply manifests/cluster-manager/hub/cluster-manager-registration-webhook-clustersetbinding-validatingconfiguration.yaml]: Got it, changed back.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**